maxim/apis/maxim_apis.py

```
@@ -689,45 +696,205 @@ def get_folders(self) -> List[Folder]:
+                    raise FileNotFoundError(f"File not found: {attachment.path}")
+                
+                size = os.path.getsize(attachment.path)
+                if size > MAX_FILE_SIZE_BYTES:
+                    raise ValueError(f"File size exceeds the maximum allowed size of {MAX_FILE_SIZE_BYTES} bytes")
+                mime_type = attachment.mime_type or 'application/octet-stream'
+                with open(attachment.path, 'rb') as f:
+                    file_data = f.read()
+            
+            return file_data, mime_type, size
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)